config/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+build_type:
+  - "cpu"
+
+pytorch_select_version:
+  - "1.0"
```

Comment:
Won't this make it so it can only be used with the cpu variant of pytorch?

Reply:
no, because we are not adding the `_pytorch_select` package as a run dependency, like in https://github.com/open-ce/torchvision-feedstock/blob/main/recipe/meta.yaml#L52.

Reply:
yea this should just make the test phase run against the pytorch cpu variant.